### PR TITLE
capstone project Week 5 

### PR DESCRIPTION
**Pull request by MicroverseNelson**
- project setup on repo
- Set up the repository and tools.
- added unique content
- chose global warming as a topic.
- Included images, text, icons, fonts.
- made the site mobile first.
- created home and about page
- included desktop version
- Adapted the 2 pages to desktop.
- added necessary interactions with Js
- added dynamic page
- Added documentation (redme file)
- code refactor

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,7 +40,7 @@
         </nav>
 
         <nav class="desktop-nav-2 flex">
-            <a href="./index.html"><img src="./capstone_assets/global4.png" alt="Logo" style="width:80px; height:80px;"></a>
+            <a href="./index.html"><img src="./capstone_assets/global4.png" alt="Logo"></a>
```
Comment (CollinsTatang) [inline index.html]: - [ ] Please kindly fix your header width to fit the [project requirement](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
### **Required Design**
header
### **Your Design**
fakeHeader

File: index.html [hunk 2]
```diff
@@ -77,31 +77,31 @@ <h3>Main Program</h3>
             <div class="program-cards flex">
                 <div class="card flex">
                     <div class="flex program-icons">
-                        <i class="fa fa-university" aria-hidden="true" style="font-size:50px;"></i>
+                        <i class="fa fa-university" aria-hidden="true"></i>
```
Comment (CollinsTatang) [inline index.html]: - [ ] Please do your best to fix the Main Program section to match the [required design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
### **Required Design**
main
### **Your Design**
mainsec

File: index.html [hunk 3]
```diff
@@ -133,24 +132,26 @@ <h3>Featured Speakers</h3>
         </div>
        <!-- <button type="button" id="moreBtn" class="speakers-button" action="">MORE <img src="./capstone_assets/dropdown.png" alt="dropdown arrow"></button> -->
     </section>
-    <a class="moreBtn" href="about.html">More <span><i class='fa fa-angle-down' style="font-size:20px;"></i></span></a>
+    <a class="moreBtn" href="about.html">More <span><i class='fa fa-angle-down'></i></span></a>
```
Comment (CollinsTatang) [inline index.html]: - [ ] Please center the speaker section to match the [required design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 
speakers

File: index.html [hunk 4]
```diff
@@ -133,24 +132,26 @@ <h3>Featured Speakers</h3>
...
-                <div class="Naver"><i class="fa fa-angellist" aria-hidden="true" style="font-size:50px;"></i></div>
-                <div class="daumkakao"><i class="fa fa-github" aria-hidden="true" style="font-size:50px;"></i></div>
-                <div class="airbnb"><i class="fa fa-twitter" aria-hidden="true" style="font-size:50px;"></i></div>
-                <div class="airbnb"><i class="fa fa-facebook" aria-hidden="true" style="font-size:50px;"></i></div>
+                <div class="mozilla"><i class="fa fa-firefox" aria-hidden="true"></i></div>
+                <div class="Google"><i class="fa fa-google" aria-hidden="true" ></i></div>
+                <div class="Naver"><i class="fa fa-angellist" aria-hidden="true" ></i></div>
+                <div class="daumkakao"><i class="fa fa-github" aria-hidden="true" ></i></div>
+                <div class="airbnb"><i class="fa fa-twitter" aria-hidden="true" ></i></div>
+                <div class="airbnb"><i class="fa fa-facebook" aria-hidden="true" ></i></div>
```
Comment (CollinsTatang) [inline index.html]: Please add the `Partner`  and `Footers` sections to your project as per the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md)
footers

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @MicroverseNelson, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Suggested changes

- [ ] Add a link to the deployed project to your README.
- [ ] Please add a video presenting your project features and a highlight about a piece of the code that you find interesting or are proud of making 

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by MicroverseNelson** [comment]
i have made the changes sir

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁 
# Hello @MicroverseNelson  😃 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

- [ ] As requested by the previous Code Reviewer, Please record a loom video and add the link in the readme file 👍🏻 

- [ ] As requested by the previous Code Reviewer, Please deploy the project and add the link to the readme file 👍🏻 

- [ ] Good Job for working on the Partners section requested by the previous code reviewer but there's still the Footer section missing. Kindly work on it. 👍🏻 

- [ ] [OPTIONAL] mobile menu shouldn't be displayed on the desktop screen. Kindly fix this 👍🏻 

Screen Shot 2022-05-28 at 8 07 29 PM

- [ ] [OPTIONAL] As per requirement, This picture should be centered align 👍🏻 

Screen Shot 2022-05-28 at 8 08 37 PM

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by RolandM99 — COMMENTED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @MicroverseNelson , 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

- [ ] Good job so far by implementing the changes requested by the previous reviewer. But, always remember to use the same branch for all requested changes asked by the previous reviewer. Never merge a PR before receive an approve from a code reviewer. When you're requested to deploy your project, don't wait to use your main branch for deployment, you can use your feature branch to deploy for a release and then, when you'll finish everything you could use the main branch to deploy.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**